app/assets/stylesheets/variables.scss

```diff
@@ -0,0 +1,5 @@
+$pink: #FFCCE7;
+$green: #DAF2DC;
+$light-blue: #81B7D2;
+$blue-purple: #4D5198;
+$font: "Abril Fatface", cursive;
```

Comment:
Could you rename `$font` to something like `$header`? That way if you want to add additional custom fonts in the future it's clear what the purpose for each one is. You could follow in another PR with this change.

Also, a new line is needed at the end of this file.

Reply:
```suggestion
$header: "Abril Fatface", cursive;
```